### PR TITLE
docs: add vercel logo

### PR DESCRIPTION
**Pull request by posthardcode**
## Purpose

Adds `Powered by Vercel` logo as required for OSS sponsorship. 

## Approach

N/A

## Testing

N/A

## Risks

N/A

### CODE REVIEW

File: templates/example.html [hunk 1]
```diff
@@ -75,5 +75,12 @@
         {{/shapes}}
       </ul>
     </section>
+    <section>
+      <a href="https://vercel.com/?utm_source=onfido-oss&utm_campaign=oss"
+        ><img
```
Comment (josokinas) [inline templates/example.html:80]: I believe it did auto-format here, however this should also be fine for Prettier, but feels a bit better:

```suggestion
      <a href="https://vercel.com/?utm_source=onfido-oss&utm_campaign=oss">
        <img
```

**Review by josokinas — APPROVED** [review]
This looks nice on a preview 👍

File: templates/example.html (hunk 1 above)
Comment (posthardcode) [reply templates/example.html]: It did, yeah. Good suggestion :)

**Comment by josokinas** [comment]
This broke because suggesstion commit format is not what our linter expects.

Please ignore, feel free to merge!

**Pull request merged**